### PR TITLE
Add SpatialRicciScalarTag

### PR DESCRIPTION
**Pull request by MarloMo**
## Proposed changes

Add SpatialRicciScalarTag and SpatialRicciScalarCompute for computing the two-dimensional intrinsic Ricci scalar of a Strahlkorper. 

### Types of changes:

- [ ] Bugfix
- [ x] New feature
- [ ] Refactor

### Component:

- [ x] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Review by geoffrey4444 — DISMISSED** [review]

File: src/ApparentHorizons/Tags.hpp [hunk 1]
```diff
@@ -344,6 +344,28 @@ struct UnitNormalVectorCompute : UnitNormalVector<Frame>, db::ComputeTag {
   using return_type = tnsr::I<DataVector, 3, Frame>;
 };
 
+/////SpatialRicciScalar is the two-dimensional intrinsic Ricci scalar curvature
```
Comment (nilsdeppe) [inline src/ApparentHorizons/Tags.hpp]: Three slashes, not five :smile_cat:

**Review by nilsdeppe — COMMENTED** [review]
Please squash the change in immediately

**Review by nilsdeppe — DISMISSED** [review]

File: src/ApparentHorizons/Tags.hpp [hunk 2]
```diff
@@ -344,6 +344,28 @@ struct UnitNormalVectorCompute : UnitNormalVector<Frame>, db::ComputeTag {
   using return_type = tnsr::I<DataVector, 3, Frame>;
 };
 
+///SpatialRicciScalar is the two-dimensional intrinsic Ricci scalar curvature
+/// of a Strahlkorper
+struct SpatialRicciScalar : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+/// Computes the two-dimensional intrinsic Ricci scalar of a Strahlkorper
```
Comment (kidder) [inline src/ApparentHorizons/Tags.hpp:353]: please add a blank line before the comment

File: src/ApparentHorizons/Tags.hpp [hunk 3]
```diff
@@ -344,6 +344,28 @@ struct UnitNormalVectorCompute : UnitNormalVector<Frame>, db::ComputeTag {
   using return_type = tnsr::I<DataVector, 3, Frame>;
 };
 
+///SpatialRicciScalar is the two-dimensional intrinsic Ricci scalar curvature
```
Comment (kidder) [inline src/ApparentHorizons/Tags.hpp]: Need a space after `///`    Also this is not the spatial Ricci scalar, so it shouldn't be called that...  isn't `StrahlkorperTags::RicciScalar`  sufficient?

**Review by kidder — CHANGES_REQUESTED** [review]

File: src/ApparentHorizons/Tags.hpp (hunk 3 above)
Comment (MarloMo) [reply src/ApparentHorizons/Tags.hpp]: Yes, that should be sufficient, I'll make the changes now.

File: src/ApparentHorizons/Tags.hpp [hunk 4]
```diff
@@ -344,11 +344,12 @@ struct UnitNormalVectorCompute : UnitNormalVector<Frame>, db::ComputeTag {
   using return_type = tnsr::I<DataVector, 3, Frame>;
 };
 
-///SpatialRicciScalar is the two-dimensional intrinsic Ricci scalar curvature
+/// Ricci scalar is the two-dimensional intrinsic Ricci scalar curvature
 /// of a Strahlkorper
```
Comment (kidder) [inline src/ApparentHorizons/Tags.hpp:348]: Sorry, I meant please rename the tags from `SpatialRicciScalar` to `RicciScalar`

**Review by kidder — CHANGES_REQUESTED** [review]

File: src/ApparentHorizons/Tags.hpp (hunk 4 above)
Comment (MarloMo) [reply src/ApparentHorizons/Tags.hpp:348]: Updated.

**Review by kidder — COMMENTED** [review]
you can squash

**Review by nilsdeppe — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Pull request merged**